src/sonic-config-engine/minigraph.py

```diff
@@ -1200,6 +1223,21 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
 
     return results
 
+def get_tunnel_entries(tunnel_intfs, lo_intfs, hostname):
+    lo_addr = ''
+    # Use the first IPv4 loopback as the tunnel destination IP
+    for addr in lo_intfs.keys():
+        if "." in addr[1]:
```

Comment:
Can we use if isinstance(obj, ipaddress.IPv4Address) to check for IPv4? Also, please have the dst_ip as IPv4 address instead of a prefix with /32 as the value.

Reply:
done